Old Pages/Turrets.md

```diff
@@ -1,28 +1,85 @@
-1.2 Heavy Turret
-1.2.1 Multiblock
-Turrets
-Turrets are multiblock structures useful for larger ships and base defense platforms. You can put them on starships and they work even while it's flying. They have more firepower than even TNT per energy bolt and can make a big difference in the hands of a skilled and well-fortified gunner. Their multiblocks are a bit larger and more complicated then usual, so the screenshots include broken-down versions
 
-To create the sign, it must be directly below (or above, if it's an upside-down turret) the center of the sponge blocks under the turret, 2 blocks down, and say the axis on the second line (e.g. "top" or "bottom"). The first line must say [turret].
+# Turrets
+Turrets are multiblock structures useful for larger ships and base defense platforms.
 
-Example: a9920a155b28436af0473c204e396c42.png
+## Using Turrets:
+- With a controller in hand, right click the turret sign.
+- Once inside, simply looking around rotates and aims the turret.
+- Left click with the controller to fire!
+- To exit, just right click.
 
-Light Turret
-Good for small or medium ships, and for precise anti-player shots, these turrets fire one single heavy energy projectile that move exactly where the player is looking.
+*Creating A Turret Sign:*
+- Must be directly below (or above) the center of the sponge blocks under the turret. 
+- 2 blocks below (or above)
+- The first line must say [turret].
+- The second line needs the axis (e.g. "top" or "bottom").
 
-Multiblock
-Materials required: 1x sponge, 1x concrete, 3x terracotta, 7x stairs, 1x iron block, 2x glass, 2x stained glass
+> Example of a top turret:
+![turretsigntop]
 
-3fe821bc1cd3c61324cfc81576e26148.png 0b8fcca833e10d2c0d73cdf770e6dc14.png 91bf873cd1666ca069b19dca75848529.png
+> Example of a bottom turret:
+![turretsignbottom]
 
-Heavy Turret
-Good for medium to large ships and for anti-ship heavy defense, these turrets are larger and slightly more expensive, but pack twice the firepower. However, they are less accurate.
+## Light Turret
+- Good for small or medium ships.
+- Precise anti-player shots.
+- Fires one projectile exactly where the player is looking.
 
-Multiblock
-Materials required: 3x sponge, 8x concrete, 13x stairs, 4x terracotta, 1x iron block, 2x glass, 4x stained glass
+![lightfront]
 
-e43952d760fe1fbed6457b8f2c366c59.png 2922733d7b5e381f0be10946dc0e58c3.png c3fcc02cab233f887acf843fa092411b.png 45449cc226e8be41eb4f82c81ec74b15.png
\ No newline at end of file
+![lightrear]
+
+### Multiblock
+***Materials required:***
+- 1x sponge,
+- 1x concrete,
+- 3x terracotta,
+- 7x stairs,
+- 1x iron block,
+- 2x glass,
+- 2x stained glass
+
+> ***Expanded view below for reference***
+
+![exlightfront]
+
+![exlightrear]
+
+## Heavy Turret
+- Good for medium to large ships
+- Anti-ship, heavy defense.
+- Larger and slightly more expensive, but pack twice the firepower. 
+- However, they are less accurate.
+
+![heavyfront]
+
+![heavyrear]
+
+### Multiblock
```

Comment:
```suggestion
### Heavy Turret Multiblock
```
this style lets it be so when i add the link button later, you can make a hyperlink to that specific section like this: https://starlegacy.net/wiki/turret#heavy-turret-multiblock